articles/azure-maps/migrate-from-bing-maps-web-app.md

```diff
@@ -41,9 +41,10 @@ If developing using a JavaScript framework, one of the following open-source pro
 
 ## Prerequisites
 
-1. Sign in to the [Azure portal](https://portal.azure.com). If you don't have an Azure subscription, create a [free account](https://azure.microsoft.com/free/) before you begin.
-2. [Make an Azure Maps account](quick-demo-map-app.md#create-an-azure-maps-account)
-3. [Obtain a primary subscription key](quick-demo-map-app.md#get-the-primary-key-for-your-account), also known as the primary key or the subscription key. For more information on authentication in Azure Maps, see [manage authentication in Azure Maps](how-to-manage-authentication.md).
+If you don't have an Azure subscription, create a [free account](https://azure.microsoft.com/free/) before you begin.
+
+1. An [Azure Maps account]
+1. A [subscription key]
```

Comment:
Change this to a bulleted list. Same for all other "Prerequisites" lists in this PR.